com.ibm.streamsx.json/info.xml

```diff
@@ -9,7 +9,7 @@
   <info:identity>
     <info:name>com.ibm.streamsx.json</info:name>
     <info:description></info:description>
-    <info:version>1.2.1.__dev__</info:version>
+    <info:version>1.2.1</info:version>
```

Comment:
Please leave the __dev__ in there, it gets replaced by the commit hash on a release build.

It also allows anyone to see if a direct codeline was being used rather than a release.